src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,12 @@
+package duke.exception;
+
+public class DukeException extends Exception{
```

Comment:
Perhaps you should put a space before this and other curly braces,
```suggestion
public class DukeException extends Exception {
```